### DISCUSSION TITLE
Updating gatsby-node.ts produces several JS alert messages

### DISCUSSION THREAD

**Discussion by AshwinSundar** [gatsbyjs/gatsby#36140] [Ideas / Feature Requests] [answered]
I am currently making some changes to gatsby-node with the server running. Any time I make a change to the file, Chrome produces an annoying alert message that moves the screen context (including keyboard context) back to Chrome from my IDE. The message appears to be produced once per change to gatsby-node.ts, so if I'm typing a few lines I end up with several alert messages that I have to click "Cancel" through before I can restart the server. Can this behavior be changed to be less disruptive? 

![Screen Shot 2022-07-15 at 10 58 27 AM](https://user-images.githubusercontent.com/13728558/179272193-9010ae21-d377-4121-92e9-0fdfff76f381.png) -> this message can show up dozens of times if I make the mistake of writing 3 lines of code in gatsby-node.ts while the server is running. Not cool!

**Comment by LekoArts** [comment, marked as answer]
You can update to `gatsby@next` or wait for the 4.19 release that gets published tomorrow (2022/07/19). There this functionality is removed.

**Reply by AshwinSundar** [reply]
Thanks for the quick fix!
